scripts/update_function_docs/update_function_docs.go

```diff
@@ -0,0 +1,294 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Usage: update_function_docs <RELEASE_BRANCH>
+//
+// e.g. update_function_docs origin/apply-setters/v0.2
```

Comment:
I'd suggest rename this file to `main.go`.
We should update it to something like `go run ./main.go origin/apply-setters/v0.2` which feels more like a script.

Reply:
I don't have a strong opinion on this, so this is okay with me. However, I was following the pattern of `generate_catalog` - so I figure we may want to keep it consistent one way or the other.

Reply:
I have done some refactoring of the module and moved the `main` function to `main.go`